### PR TITLE
🐛 Fixes related to release capm3/ipam 1.7 and bmo 0.6.0

### PR DESCRIPTION
**Pull request by adilGhaffarDev**
New branches were put under the wrong prow config. This PR is fixing that.

### CODE REVIEW

File: prow/manifests/overlays/metal3/config.yaml [hunk 1]
```diff
@@ -415,6 +415,7 @@ presubmits:
             imagePullPolicy: Always
     - name: gomod
       branches:
+        - release-0.6
```
Comment (tuminoid) [inline prow/manifests/overlays/metal3/config.yaml:418]: Put main on top please on each of these. We'll add the releases beneath as needed.

Comment (adilGhaffarDev) [reply prow/manifests/overlays/metal3/config.yaml]: done

**Review by tuminoid — COMMENTED** [review]
/approve

**Pull request merged**